server/graphql/v2/object/Host.js

```diff
@@ -64,6 +67,46 @@ const getNumberOfDays = (startDate, endDate, host) => {
   return days(since, endDate || undefined);
 };
 
+const convertCurrencyAmount = async (fromCurrency, toCurrency, date, amount) => {
+  const fxRate = await getFxRate(fromCurrency, toCurrency, date);
+  const convertedAmount = Math.round(amount * fxRate);
+  return { date: date, amount: convertedAmount };
+};
```

Comment:
This is a bit reinventing the wheel, `convertToCurrency` in `lib/currency` should be enough?